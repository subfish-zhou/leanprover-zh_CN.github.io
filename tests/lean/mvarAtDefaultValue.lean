structure A where
  x : Nat

structure B extends A where
  x := _ -- should be error

structure C extends A where
  x := _ + 1 -- should be error
