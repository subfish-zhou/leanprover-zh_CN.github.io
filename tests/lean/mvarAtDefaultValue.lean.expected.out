mvarAtDefaultValue.lean:5:2-5:3: error: invalid default value for field, it contains metavariables
  ?m
mvarAtDefaultValue.lean:8:2-8:3: error: invalid default value for field, it contains metavariables
  ?m + 1
